assets/src/edit-story/components/panels/animation/effectChooser.js

```diff
@@ -74,80 +109,239 @@ const GridItemHalfRow = styled(GridItem)`
+              direction: DIRECTION.LEFT_TO_RIGHT,
+            })
+          }
+        >
+          <ContentWrapper>{__('Woosh In', 'web-stories')}</ContentWrapper>
+          <WhooshInLeftAnimation>
+            {__('Woosh In', 'web-stories')}
+          </WhooshInLeftAnimation>
+        </GridItemHalfRow>
+        <GridItemHalfRow
+          aria-label={__('Whoosh In from Right Effect', 'web-stories')}
+          onClick={() =>
+            onAnimationSelected({
+              animation: ANIMATION_EFFECTS.WHOOSH_IN.value,
+              direction: DIRECTION.RIGHT_TO_LEFT,
+            })
+          }
+        >
+          <WhooshInRightAnimation>
+            {__('Woosh In', 'web-stories')}
+          </WhooshInRightAnimation>
+          <ContentWrapper>{__('Woosh In', 'web-stories')}</ContentWrapper>
+        </GridItemHalfRow>
+        <GridItem
+          aria-label={__('Pan from Left Effect', 'web-stories')}
+          onClick={() =>
+            onAnimationSelected({
+              animation: ANIMATION_EFFECTS.PAN.value,
+              direction: DIRECTION.LEFT_TO_RIGHT,
+            })
+          }
+        >
+          <ContentWrapper>{__('Pan', 'web-stories')}</ContentWrapper>
+          <PanLeftAnimation>{__('Pan', 'web-stories')}</PanLeftAnimation>
+        </GridItem>
+        <GridItem
+          aria-label={__('Pan from Top Effect', 'web-stories')}
+          onClick={() =>
+            onAnimationSelected({
+              animation: ANIMATION_EFFECTS.PAN.value,
+              direction: DIRECTION.TOP_TO_BOTTOM,
+            })
+          }
+        >
+          <ContentWrapper>{__('Pan', 'web-stories')}</ContentWrapper>
+          <PanTopAnimation>{__('Pan', 'web-stories')}</PanTopAnimation>
+        </GridItem>
+        <GridItem
+          aria-label={__('Pan from Bottom Effect', 'web-stories')}
+          onClick={() =>
+            onAnimationSelected({
+              animation: ANIMATION_EFFECTS.PAN.value,
+              direction: DIRECTION.BOTTOM_TO_TOP,
+            })
+          }
+        >
+          <ContentWrapper>{__('Pan', 'web-stories')}</ContentWrapper>{' '}
+          <PanBottomAnimation>{__('Pan', 'web-stories')}</PanBottomAnimation>
+        </GridItem>
+        <GridItem
+          aria-label={__('Pan from Right Effect', 'web-stories')}
+          onClick={() =>
+            onAnimationSelected({
+              animation: ANIMATION_EFFECTS.PAN.value,
+              direction: DIRECTION.RIGHT_TO_LEFT,
+            })
+          }
+        >
+          <ContentWrapper>{__('Pan', 'web-stories')}</ContentWrapper>
+          <PanRightAnimation>{__('Pan', 'web-stories')}</PanRightAnimation>
+        </GridItem>
+        <GridItemHalfRow
+          aria-label={__('Zoom In Effect', 'web-stories')}
+          onClick={() =>
+            onAnimationSelected({
+              animation: ANIMATION_EFFECTS.ZOOM.value,
+              defaultValue: 0,
+            })
```

Comment:
We may want to alter the input on these a little down the road by passing in a `zoomFrom` & `zoomTo` values, but this works for now and we can alter these to our needs when we pull the effect chooser into the app